### PR TITLE
clone 2.1.2: Fixed missing parameters and overload.

### PR DESCRIPTION
**Pull request by DG-za**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/pvorb/clone
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/clone/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for clone 0.1.11
+// Type definitions for clone 2.1.2
 // Project: https://github.com/pvorb/node-clone
-// Definitions by: Kieran Simpson <https://github.com/kierans/DefinitelyTyped>
+// Definitions by: DG-za <https://github.com/DG-za/DefinitelyTyped>
```
Comment (peterblazejewicz) [inline types/clone/index.d.ts:3]: So this seems wrong,
I think the original owner line should be left and look like:
```suggestion
// Definitions by: Kieran Simpson <https://github.com/kierans>
```
you should add your line, but change it to:
```ts
DG-za <https://github.com/DG-za>
```

File: types/clone/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for clone 0.1.11
+// Type definitions for clone 2.1.2
```
Comment (peterblazejewicz) [inline types/clone/index.d.ts:1]:
```suggestion
// Type definitions for clone 2.1
```
https://github.com/definitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
@DG-za
Looks legit, please review my comments, thx!

File: types/clone/index.d.ts (hunk 2 above)
Comment (DG-za) [reply types/clone/index.d.ts]: Thanks, will keep that in mind next time!

File: types/clone/index.d.ts (hunk 1 above)
Comment (DG-za) [reply types/clone/index.d.ts]: Fixed and added my line as a second owner.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@DG-za thx for the contribution!

**Review by rbuckton — APPROVED** [review]

**Comment by DG-za** [comment]
Ready to merge

**Pull request merged**